distribution/test/itests/test-itests-ddf/pom.xml

```diff
@@ -337,7 +337,36 @@
                                     <asyncDestroyOnShutdown>true</asyncDestroyOnShutdown>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>attach-ace-debugger</id>
+                                <phase>pre-integration-test</phase>
```

Comment:
❓ Will this fail the build if we only run `pre-integration-test`? What if we use the skip itests flag while doing a clean install?

I have reservations about using maven plugins like this for tests. It feels similar to some of the problems we've ran into running solr through the exec-plugin. Did you look have the debugger executed by pax exam before the test starts?

Reply:
The whole plugin gets skipped from the configuration above on line 318